tests/thread_commission_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/canonical/matter-snap-testing/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	otbrSnap = "openthread-border-router"
+
+	defaultInfraInterfaceValue = "wlan0"
+	infraInterfaceKey          = "infra-if"
+	localInfraInterfaceEnv     = "LOCAL_INFRA_IF"
+)
+
+func TestAllClustersAppThread(t *testing.T) {
```

Comment:
This test should not be run on GH actions, as it always fails due to the missing remote machine and RCPs dependencies.